azure-pipelines.yml

```diff
@@ -5,24 +5,60 @@ jobs:
   timeoutInMinutes: 360
   strategy:
     matrix:
-      cos7_x86_64:
+      cos7_x86_64_1:
         DISTARCH: 'centos7-x86_64'
-      cos7_aarch64:
+        PTOP: '1:4'
+      cos7_x86_64_2:
+        DISTARCH: 'centos7-x86_64'
+        PTOP: '2:4'
+      cos7_x86_64_3:
+        DISTARCH: 'centos7-x86_64'
+        PTOP: '3:4'
+      cos7_x86_64_4:
+        DISTARCH: 'centos7-x86_64'
+        PTOP: '4:4'
```

Comment:
I don't think we should do this TBH, mostly because even a full cos7 rebuilds work most of the time (and a restart only has to pick up the delta), but more importantly, this will almost certainly lead to missing dependencies in some jobs (e.g. pam-devel has ~7 other CDTs it depends on, which will almost certainly not all be hashed into one specific job.

Reply:
I put in the fix. Hopefully this makes the builds more reliable.